wireguard install - scripts/basic/fixdep not found

clean ubuntu, gcc and build-essentials installed

```
➜  ~docker run --rm -it \
        --name wireguard \
        -v /lib/modules:/lib/modules \
        -v /usr/src:/usr/src:ro \
        r.j3ss.co/wireguard:install
Building the wireguard kernel module...
getconf: LFS_CFLAGS: unknown variable
getconf: LFS_LDFLAGS: unknown variable
getconf: LFS_LIBS: unknown variable
  CC [M]  /wireguard/src/main.o
/bin/sh: scripts/basic/fixdep: not found
make[2]: *** [scripts/Makefile.build:334: /wireguard/src/main.o] Error 127
make[1]: *** [Makefile:1552: _module_/wireguard/src] Error 2
make: *** [Makefile:36: module] Error 2
```

```
➜  ~ ls /usr/src
fuse-3.0.0               linux-headers-4.13.0-31-generic  linux-headers-4.15.0-33
fuse-3.0.0.tar.gz        linux-headers-4.15.0-32          linux-headers-4.15.0-33-generic
linux-headers-4.13.0-31  linux-headers-4.15.0-32-generic  s3fs-fuse
➜  ~ uname -r
4.15.0-33-generic
```

--- a/wireguard/install/Dockerfile
+++ b/wireguard/install/Dockerfile
@@ -15,6 +15,7 @@ RUN apk add --no-cache \
 	build-base \
 	ca-certificates \
 	elfutils-libelf \
+	libelf-dev \
 	libmnl-dev
 
 # https://git.zx2c4.com/WireGuard/refs/

--- a/wireguard/install/Dockerfile
+++ b/wireguard/install/Dockerfile
@@ -14,7 +14,7 @@ LABEL maintainer "Jessie Frazelle <jess@linux.com>"
 RUN apk add --no-cache \
 	build-base \
 	ca-certificates \
-	libelf-dev \
+	elfutils-libelf \
 	libmnl-dev
 
 # https://git.zx2c4.com/WireGuard/refs/

